live-examples/js-examples/array/meta.json

```diff
@@ -48,6 +48,12 @@
             "title": "JavaScript Demo: Array.find()",
             "type": "js"
         },
+        "arrayGroupBy": {
+            "exampleCode": "./live-examples/js-examples/array/array-groupby.js",
+            "fileName": "array-groupby.html",
+            "title": "JavaScript Demo: Array.groupBy()",
+            "type": "js"
+        },

```

Comment:
Oh, I just realised that the entries here should be alphabetised (so `arrayGroupBy` should go after `arrayFrom`). I wish we had linting for that.

Also, I'm really sorry to give you feedback in bits and pieces :(.